contentful.json

```diff
@@ -10,7 +10,7 @@
     "On Premises Installation": "No",
     "Cloud Service Hosted in Europe": "No",
     "Commercial Support Available?": "Yes",
-    "GraphQL API": "No",
+    "GraphQL API": "Yes",
```

Comment:
https://www.contentful.com/developers/docs/references/graphql/

Reply:
Nice, I only knew about the wrapper implementation (cf-graphql).